logging/measure/model/apitrace/apitrace.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+// Package apitrace provides data model for binary API trace informations.
+package apitrace;
+
+message Trace {
+    message TracedEntry {
+        uint32 index = 1;                       // Trace index
+        string msg_name = 2;                    // Message name (or any other string)
+        string duration = 3;                    // Duration in milliseconds
```

Comment:
Not sure if it's good idea to keep duration as string. It's gonna make it harder for parse the value for external systems. It should be integer, perhaps with unit as part of name so it's clear to readers.

Reply:
true, but I would keep both formats, the integer which is easy to parse, and keep also the sting which is easier to read

Reply:
I'd strongly oppose this. It might easily cause problems when these two values are not in sync and also very likely to confuse readers. 

I think consumers of this data are gonna do what they need with the value anyway, but generally we should provide low level (unsigned) integer value which is also easily comparable.

Reply:
Perhaps this might be useful? https://godoc.org/github.com/gogo/protobuf/types#Duration